### PR TITLE
Update with the Polymer 1.1 style recommendations

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "menu"
   ],
   "main": "paper-menu.html",
-  "private": true,
```
Comment (samccone) [inline bower.json:15]: do we want to remove this¿

Comment (abdonrd) [inline bower.json:15]: @samccone Why private if it is a public repository?

**Comment by abdonrd** [comment]
Updated @samccone!

File: paper-menu.html [hunk 2]
```diff
@@ -64,43 +65,32 @@
...
       <content></content>
     </div>
-
   </template>
 
-</dom-module>
-
-<script>
-
-(function() {
```
Comment (samccone) [inline paper-menu.html:92]: why drop the closure here?

Comment (abdonrd) [inline paper-menu.html:92]: Sorry, updated!

**Comment by samccone** [comment]
lgtm :+1:

**Comment by notwaldorf** [comment]
Thanks!

**Pull request merged**